app/customer.py

```diff
@@ -0,0 +1,50 @@
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, name: str, product_cart: dict,
+                 location: list, money: float, car: Car) -> None:
```

Comment:
The variable name 'product_cart' could be more descriptive. Consider renaming it to something like 'shopping_cart' for better clarity.